post-install.yml

```diff
@@ -9,20 +9,10 @@
     vagrant_user: "{% if ansible_os_family == 'RedHat' %}vagrant{% else %}ubuntu{% endif %}"
 
   tasks:
-    # Some tomfoolery to get the demo module to install.
-    # Subject to https://www.drupal.org/node/2599228
-    # Should be fixed in Drupal 8.7
-    - name: Install islandora_defaults module (fail ok)
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y en islandora_defaults"
-      ignore_errors: yes
-    - name: Update entities
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y entity:updates"
-    - name: Uninstall islandora_defaults module
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y pmu islandora_defaults"
-    - name: Install islandora_defaults module (should not fail)
+
+    - name: Install islandora_defaults module
       command: "{{ drush_path }} --root {{ drupal_core_path }} -y en islandora_defaults"
-    - name: Import feature
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y fim --bundle=islandora islandora_defaults"
```

Comment:
Re: @seth-shaw-unlv's comments I also got the same behaviour. I re-added this "Import feature" and everything seemed to come up hunky-dorry.